Dockerfile

```diff
@@ -53,20 +51,14 @@ RUN wget --quiet "${JDK_URL}" && \
 ENV JAVA_HOME="/jdk-12" 
 ENV PATH="$JAVA_HOME/bin:$PATH"
 
-#Install wkhtmltopdf for PDF Reports
-WORKDIR /tmp
-RUN wget --quiet "${WKH_URL}" && \
-    tar xJf "${WKH_FILE}"
-WORKDIR /tmp/wkhtmltox/bin
-RUN cp ./* /usr/bin
-WORKDIR /tmp
-RUN rm -rf ./wkhtmltox && \
-    rm -f "${WKH_FILE}"
-
 #Add MobSF master
 COPY . /root/Mobile-Security-Framework-MobSF
 WORKDIR /root/Mobile-Security-Framework-MobSF
 
+#Symlink wkhtml tools
+RUN ln -s /root/Mobile-Security-Framework-MobSF/StaticAnalyzer/tools/wkhtmltox/bin/wkhtmltopdf /usr/bin/ \
+    ln -s /root/Mobile-Security-Framework-MobSF/StaticAnalyzer/tools/wkhtmltox/bin/wkhtmltoimage /usr/bin/
+
```

Comment:
@superpoussin22 Is line 60 needed. We only want wkhtmltopdf?

Also why not 

```
RUN ln -s /root/Mobile-Security-Framework-MobSF/StaticAnalyzer/tools/wkhtmltox/bin/wkhtmltopdf /usr/bin/wkhtmltopdf
```

Reply:
will remove the unneeded binary